helpers/devices/devices.py

```diff
@@ -3,12 +3,12 @@
 from api.devices.schema import Devices as DeviceSchema
 
 
-def update_device_last_seen(info, room_id, check_in_time):
+def update_device_last_seen(info, room_id, new_last_seen):
```

Comment:
Let's change this variable name to something more meaningful. It needs to be clear that `new_last_seen` is a time value. So call it something like `time` or `activity_time`

Reply:
I am going to change it. I thought about naming it 'time' and I thought that maybe it could be confusing.